### DISCUSSION TITLE
Option to display the list like the default quick switcher (only the path on a single line)

### DISCUSSION THREAD

**Discussion by ToNyRANDRIAMANANTSOA** [darlal/obsidian-switcher-plus#176] [General]
Hello,

I enjoy the added features of switcher plus, but I am used to the more compact interface of the list like how the default core quick switcher plugin display it with only the path on a single line, i.e. instead of the filename and file path below.

Is there a way I can keep it this way?
(NB: I saw that the files which are not created are already displayed like that in this plugin)

Example:
![image](https://github.com/user-attachments/assets/4c7b543c-3c3c-4534-8a1c-9901ecb53705)

Instead of:
![image](https://github.com/user-attachments/assets/ba9c3133-b875-45de-969a-5f091ef93529)

Thanks.

**Comment by darlal** [comment]
Interestingly Switcher++ originally displayed path and filename on one line as you're describing. But there was a feature request to break it up on separate lines see #51. 

Two suggestions to try: 
1) If the path is not useful to you, you could actually hide it altogether by changing the `Preferred file path display format` [setting](https://github.com/darlal/obsidian-switcher-plus?tab=readme-ov-file#general-settings). Though if the path is important to you then this won't be helpful

2) If you're comfortable with CSS, you could probably rearrange the display and move it all to one line by modifying the `styles.css` file in the plugin directory. You could also use this to make display more compact in general if you'd like.

Thanks!

**Comment by darlal** [comment]
Actually looking at the screenshots more closely, in both cases (this one, and in #180 from @itsonlyjames) you're in Standard mode which already has an override switch in the settings. 

This can be achieved by disabling the setting [Override Standard mode behavior](https://github.com/darlal/obsidian-switcher-plus?tab=readme-ov-file#general-settings). That should stop the Obsidian single line rendering in Standard mode from being overridden. Thanks!

**Comment by itsonlyjames** [comment]
Close but no cigar. 

The benefit of using override standard mode behaviour specifically for me is that it doesn't duplicate tab opening, as in, if a tab is already open then it switches to that tab instead of opening a new one. 

Plus, what other functionality is lost by disabling this setting?

This functionality is paramount to my usage of the plugin, however, the current display being multi-line is not ideal. A perfect solution would be to seperate out functionality and design differences to allow for either/both.

**Comment by churnish** [comment]
I support the request for a single line file path display option. Would make things look cleaner.

**Comment by darlal** [comment]
[Release 5.0.0](https://github.com/darlal/obsidian-switcher-plus/releases/tag/5.0.0) introduces a separate [Override Standard mode rendering](https://github.com/darlal/obsidian-switcher-plus/tree/5.0.0?tab=readme-ov-file#general-settings) setting for the rendering. Thanks!
